src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const rules = sourceString.split(';');
+
+  const result = {};
+
+  rules.forEach((rule) => {
+    const ruleCopy = rule.trim();
+
+    if (ruleCopy !== '') {
+      const [key, value] = ruleCopy.split(':').map((item) => item.trim());
+
+      result[key] = value;
+    }
+  });
```

Comment:
[CODE KNOWLEDGE] - Don't use iteration methods to modify an already existing array/object. These methods return a new array, so you should use it. Instead of using `forEach` to directly modify the `result` object, consider using `reduce` to return a new object with the accumulated styles.